### PR TITLE
Correct layerInfo on mobile due to react-ui update.

### PR DESCRIPTION
**Pull request by edellucien**

### CODE REVIEW

File: src/components/Dialog/Dialog.scss [hunk 1]
```diff
@@ -81,25 +81,30 @@
...
-    left: 10px;
-    right: 10px;
-    width: calc(100% - 20px);
-    height: calc(40% - 10px);
-    max-height: 50%;
-    margin: auto;
-    transform: none !important; // block drag
+  .wkp-dialog {
+    .tm-rnd {
+      top: unset !important;
```
Comment (edellucien) [inline src/components/Dialog/Dialog.scss]: forced to use '!important' to overwrite the new lib 'react-rnd' from react-ui new version.

**Comment by oterral** [comment]
@edellucien conflict

**Comment by edellucien** [comment]
Temporary remove cypress action, due to a upflow github action problem.

**Comment by edellucien** [comment]
> Temporary remove cypress action, due to a upflow github action problem.

Need to use ubuntu-16.04 as github action env to fix the cypress action.

**Pull request merged**